worlds/AutoWorld.py

```diff
@@ -412,6 +413,16 @@ def get_data_package_data(cls) -> "GamesPackage":
         res["checksum"] = data_package_checksum(res)
         return res
 
+    # allows modification of webhost during startup, this is run once
+    @classmethod
+    def run_webhost_setup(cls):
```

Comment:
wouldn't it make more sense to make these part of webworld?

Reply:
Maybe. Currently, webworld is just a fancy typeddict, without any methods